program/Cargo.toml

```diff
@@ -14,14 +14,14 @@ bytemuck = { version = "1.16.1", features = ["derive"] }
 num_enum = "0.7.2"
 paladin-stake-program = { git = "https://github.com/paladin-bladesmith/stake-program.git" }
 shank = "0.4.2"
-solana-program = "2.0.2"
+solana-program = "2.0"
 spl-discriminator = "0.3.0"
 spl-program-error = "0.5.0"
 
 [dev-dependencies]
 proptest = "1.5.0"
-solana-program-test = "2.0.2"
-solana-sdk = "2.0.2"
+solana-program-test = "2.0"
+solana-sdk = "2.0"
```

Comment:
Were these changes intentional?

Reply:
Yeah, We have `spl-pod` dependent on `swolana-program = "2.0.3"` in the stake program, and that means anything below that won't be tolerated by Cargo. So to avoid any other dependency issue, I let Cargo sort out the patch versions here.